bundle/root_test.go

```diff
@@ -102,3 +103,32 @@ func TestRootLookupError(t *testing.T) {
 	_, err := mustGetRoot()
 	require.ErrorContains(t, err, "unable to locate bundle root")
 }
+
```

Comment:
Can you add a test case where you have bundle.yml in the temp dir and also have ExtraInclude set and make sure it loads the config correctly?